doc/source/dev/governance/governance.rst

```diff
@@ -378,6 +378,22 @@ A list of current Institutional Partners is maintained at the page
 :ref:`governance-people`.
 
 
+Commit Rights in the NumPy Code Base
+====================================
+
+The NumPy Steering Council can grant and revoke commit rights to the NumPy code
+repository, currently hosted at `github`_.
+
+The policy for granting commit priveleges is based on trust, with the
```

Comment:
typo "privileges" (and typo in same word again 2 lines lower)

Reply:
fixed